components/mgmtworker/scripts/start.py

```diff
@@ -10,7 +10,31 @@
 import utils  # NOQA
 
 MGMT_WORKER_SERVICE_NAME = 'mgmtworker'
+CELERY_PATH = '/opt/mgmtworker/env/bin/celery'  # also hardcoded in create
+
+
+@utils.retry(ValueError)
```

Comment:
`ValueError` isn't the ideal choice, but it's the closest of built-in exceptions IMO - the only real alternative being `RuntimeError` which is too general i think. And the exception doesn't leave this module anyway, so it's not a big problem.